src/main/java/winstone/Launcher.java

```diff
@@ -442,15 +449,23 @@ public static void initLogger(Map<String, String> args) throws IOException {
         // boolean showThrowingLineNo = Option.LOG_THROWING_LINE_NO.get(args);
         boolean showThrowingThread = Option.LOG_THROWING_THREAD.get(args);
         OutputStream logStream;
+        Charset logCharset;
         if (args.get("logfile") != null) {
-            logStream = new FileOutputStream(args.get("logfile"));
+            try {
+                logStream = Files.newOutputStream(Paths.get(args.get("logfile")), StandardOpenOption.CREATE, StandardOpenOption.APPEND);
```

Comment:
Technically appending to an existing log file is a change in behavior, but I think it makes more sense to append. In `jenkinsci/packaging` it is currently recommended to set the `-logfile` option to `/var/log/jenkins/jenkins.log`, where it will be rotated by `logrotate(8)`.

Reply:
as it's a breaking change the PR should have the appropriate label...

Reply:
> as it's a breaking change the PR should have the appropriate label...

I updated the label and the changelog to mention this change.